netbout-web/src/test/java/com/netbout/rest/MarkdownTest.java

```diff
@@ -162,6 +162,20 @@ public void formatsBulletsToHtml() throws Exception {
     }
 
     /**
+     * Markdown can break a single line.
+     * @throws Exception If there is some problem inside
```

Comment:
@krzyk As you can see, all other methods from test are declaring that it throws exception, but none of them throw it. What I understood that this is done by design. Am I right @dmzaytsev?